guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<p>The GeoEnrichment module provides a python interface for accessing the demographic data provided through Business Analyst, allowing you to enrich study areas, access standard geographies, and create reports. Accessing standard geographies enables the retrieving and enriching of standard jurisdictional areas, such as counties, postal (zip) codes, or US Census Block Groups in the United States.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>